package/boot/uboot-envtools/files/mediatek_filogic

```diff
@@ -101,6 +101,7 @@ glinet,gl-mt6000)
 glinet,gl-mt3000)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x80000" "0x20000"
 	;;
+buffalo,wsr-6000ax8|\
```

Comment:
Please sort alphabetically. (this block should be placed before `cmcc,rax3000m`)

Reply:
Thank you for pointing that out. I've fixed it.